_i18n/fr/get-started/contributing.md

```diff
@@ -35,7 +35,7 @@ L'extraction minière garantit que le réseau reste sécurisé et décentralisé
                                 </div>
                             </div>
                             <div class="row start-xs">
-                                <p>Monero utilise un <a href="https://forum.getmonero.org">forum funding system</a> au travers duquel des projets sont proposé au développement et financés par la communauté. Le financement est gardé en entiercement et rémunèrent les développeurs lorsque des jalons de programmation sont atteints. Chacun peut faire de nouvelle proposition ou en financer d'existantes.</p>
+                                <p>Monero utilise un <a href="https://forum.getmonero.org">forum funding system</a> au travers duquel des projets sont proposé au développement et financés par la communauté. Le financement est entiercé et rémunère les développeurs lorsque des jalons de programmation sont atteints. Chacun peut faire de nouvelles propositions ou en financer d'existantes.</p>
```

Comment:
sont proposé -> proposés

Reply:
Maybe add a translation for forum funding system? Système de financement par la communauté?

Reply:
ok forthe first. I tried to find something for FFS that could be close enough... but if you're ok with it and @erciccione is also, i'll do it.